bin/compile

```diff
@@ -47,13 +47,15 @@ else
   CRYSTAL_VERSION_REASON='due to latest release at https://github.com/crystal-lang/crystal'
 fi
 
-CRYSTAL_URL="https://github.com/crystal-lang/crystal/releases/download/$CRYSTAL_VERSION/crystal-$CRYSTAL_VERSION-1-linux-x86_64.tar.gz"
+CRYSTAL_RELEASE_ID=`${CURL} https://api.github.com/repos/crystal-lang/crystal/releases/tags/${CRYSTAL_VERSION} | json_value id 1`
+
+CRYSTAL_URL=`${CURL} https://api.github.com/repos/crystal-lang/crystal/releases/${CRYSTAL_RELEASE_ID}/assets | grep 'https://github.com/crystal-lang/crystal/releases/download/.*-linux-x86_64.tar.gz' | sed 's/.*: "\(.*\)"/\1/'`
```

Comment:
why do we need to get the assets of a release? the first request already seems to respond all the published assets.

Am I missing something or one request is enough?

```
CRYSTAL_URL=`${CURL} https://api.github.com/repos/crystal-lang/crystal/releases/tags/${CRYSTAL_VERSION} | grep 'https://github.com/crystal-lang/crystal/releases/download/.*-linux-x86_64.tar.gz' | sed 's/.*: "\(.*\)"/\1/'`
```